windows_docker_resources/install_ros2_rolling.json

```diff
@@ -2,6 +2,7 @@
   "ros2_windows": {
     "download_sources": "false",
     "vs_version": "buildtools",
+    "vs_release": "2022",
```

Comment:
If we wanted to merge this change before locking in the VS2022 switch we could merge this leaving the rolling default at 2019.